HULKs/hulk
# Add new behavior test files

## Introduced Changes

- Add new scenario test files for the behavior simulator
- Delete old behavior test files

## ToDo / Known Issues

None

## Ideas for Next Iterations (Not This PR)
None

## How to Test

Run the scenario files in the behavior simulator and check if it works the way the filename suggests.
